paymentsheet/src/main/java/com/stripe/android/paymentelement/embedded/content/EmbeddedSelectionChooser.kt

```diff
@@ -95,7 +95,11 @@ internal class DefaultEmbeddedSelectionChooser @Inject constructor(
         previousSelection: PaymentSelection.New,
         paymentMethodMetadata: PaymentMethodMetadata,
     ): Boolean {
-        val newFormType = formHelperFactory.create(coroutineScope, paymentMethodMetadata) {}
+        val newFormType = formHelperFactory.create(
+            coroutineScope,
+            paymentMethodMetadata,
+            eventReporter = null
```

Comment:
I don't think we should pass null, and I also don't think it should be nullable.

Why is it nullable?